samples/v1beta1/buildstrategy/multiarch-native-buildah/buildstrategy_multiarch_native_buildah_cr.yaml

```
@@ -0,0 +1,544 @@
+---
+apiVersion: shipwright.io/v1beta1
+kind: ClusterBuildStrategy
+metadata:
+  name: multiarch-native-buildah
+spec:
+  volumes:
+    - name: oci-archive-storage
+      emptyDir: { }
```

If you want developers to cache container image layers, you should also add the `overridable: true` field so a `Build` or `BuildRun` can change this to be a PVC.
```suggestion
    - name: oci-archive-storage
      emptyDir: {}
      overridable: true
```